packages/cron/src/jobs/storage.js

```diff
@@ -179,11 +193,23 @@ export async function checkStorageUsed ({ roPg, emailService, userBatchSize = 20
       throw new Error(`${supportEmail} does not exists`)
     }
 
+    const usersOverQuotaAugmented = await Promise.all(
+      usersOverQuota.map(async u => {
+        const { rows } = await roPg.query(GET_DATE_LATEST_UPLOAD_OR_PSA_REQUEST, [
```

Comment:
This might become a bit slow.  Could bulk fetch `GET_DATE_LATEST_UPLOAD_OR_PSA_REQUEST` with a list of user IDs instead of a single ID and then merge.

Reply:
Good point.
However this list **should** never grow, the idea is that users are disabled once they over quota and they wouldn't appear in that list anymore. 
So not really sure is worth "complicating" this? What do you think?

Reply:
Agreed it is fine as it is 👍